import pytest


@pytest.mark.failing
def test_failing():
    assert 1 == 2